### ISSUE TITLE
shrinkwrap not same as the package.json dependencies

### ISSUE THREAD

**Issue by adnan-i** [nodemailer/nodemailer#613]
nodemailer: 2.4.0
node: 0.12.7
OS: Mac OS Yosemite

I was trying to install the latest nodemailer, and the installation was success, but then `npm shrinkwrap` complaints about nodemailer's dependencies:
```
npm ERR! invalid: have mailcomposer@3.7.0 (expected: 3.9.0) /project-path/node_modules/nodemailer/node_modules/mailcomposer
npm ERR! invalid: have nodemailer-direct-transport@3.0.7 (expected: 3.1.0) /project-path/node_modules/nodemailer/node_modules/nodemailer-direct-transport
npm ERR! invalid: have nodemailer-shared@1.0.4 (expected: 1.0.5) /project-path/node_modules/nodemailer/node_modules/nodemailer-shared
npm ERR! invalid: have nodemailer-smtp-pool@2.5.2 (expected: 2.6.0) /project-path/node_modules/nodemailer/node_modules/nodemailer-smtp-pool
npm ERR! invalid: have nodemailer-smtp-transport@2.4.2 (expected: 2.5.0) /project-path/node_modules/nodemailer/node_modules/nodemailer-smtp-transport
npm ERR! extraneous: mailcomposer@3.7.0 /project-path/node_modules/nodemailer/node_modules/mailcomposer
npm ERR! extraneous: nodemailer-direct-transport@3.0.7 /project-path/node_modules/nodemailer/node_modules/nodemailer-direct-transport
npm ERR! extraneous: nodemailer-smtp-pool@2.5.2 /project-path/node_modules/nodemailer/node_modules/nodemailer-smtp-pool
npm ERR! extraneous: nodemailer-smtp-transport@2.4.2 /project-path/node_modules/nodemailer/node_modules/nodemailer-smtp-transport
```

Your package.json asks for `mailcomposer@3.9.0` but your shrinkwrap file asks for `mailcomposer@3.7.0`. Aren't these two supposed to define the same exact dependency version? How do I fix this?

**Comment by MauriceSchoenmakers** [comment]
+1

**Comment by andris9** [comment]
Oops, forgot to update it. Fixed in Nodemailer@2.4.1
